icebox/icebox.py

```diff
@@ -1500,6 +1546,12 @@ def parse_db(text, device="1k"):
         "S0":   ( 23, 0, "fabout" ),
         "S1":   ( 24, 0, "fabout" ),
     },
+    "lm4k": {
```

Comment:
It looks like this is good, but ISTR seeing a Lattice note saying there was no WARMBOOT on the LM series. Maybe that was wrong though